app/src/main/java/com/example/unlibrary/library/LibraryBookDetailsFragment.java

```diff
@@ -65,6 +68,25 @@ public View onCreateView(@NotNull LayoutInflater inflater, ViewGroup container,
                     .show();
         });
 
+        //Setup the list of requesters
```

Comment:
Mega super nit: IMO I prefer a space before the text of a comment.

```suggestion
        // Setup the list of requesters
```